scripts/install/functions.sh

```diff
@@ -242,6 +242,7 @@ function install_from_sources {
         'smartedit'
         'cds'
         'cdc'
+        'epd-visualization'
```

Comment:
I've run this test installation script, but EPD Visualisations feature library is not installed. This line only causes the '@spartacus/epd-visualisation' to be published to a local npm registry (verdaccio). But we need to also add a logic to install this feature in the fresh app.

Reply:
You might take as an example the CDC or CPQ.
- Create a variable eg. `$ADD_EPD_VISUALISATION` set to false in config.default.sh.
- create a function `add_epd_visualisation`
```bash
function add_epd_visualisation {
    if [ "$ADD_EPD_VISUALISATION" = true ] ; then
        ng add --skip-confirmation @spartacus/epd-visualisation@${SPARTACUS_VERSION} --baseUrl="${EPD_VISUALISATION_BASE_URL}" --interactive false
    fi
}
```
... and set the variable in `config.default.sh`:
```bash
EPD_VISUALISATION_BASE_URL="https://epd-acc-eu20-consumer.epdacc.cfapps.eu20.hana.ondemand.com"
```
- run this function `add_epd_visualisation` inside: `add_spartacus_csr`, `add_spartacus_ssr`, `add_spartacus_ssr_pwa`

Reply:
I will check this.

Reply:
I have updated the script.